std-bits/snowflake/src/main/java/org/enso/snowflake/SnowflakeIntegerColumnMaterializer.java

```diff
@@ -119,10 +119,26 @@ public Storage<?> seal() {
   @Override
   public StorageType getType() {
     // The type of the builder can change over time, so we do not report any stable type here.
-    // Same as in InferredBuilder.
     return null;
   }
 
+  @Override
+  public void copyDataTo(Object[] items) {
+    if (currentSize > 0) {
+      if (mode == Mode.LONG) {
+        for (int i = 0; i < currentSize; i++) {
+          if (intsMissing.get(i)) {
```

Comment:
```suggestion
         items[i] = intsMissing.get(i) ? null : ints[i];
```

A nit but the deep nesting is bothering me. Feel free to ignore

Reply:
assuming the build passes will put in next PR, otherwise will add to this one.